File: ex06/ex06_10/ex06_10.js
import promptSync from 'prompt-sync';
const prompt = promptSync();
function add(arr) {
  arr[0] += arr[1]; 
}
function multiply(arr) {
  arr[0] *= arr[1];
}
function main() {
  let a = [];
  let b = [];
  a[0] = Number(prompt(" a = "));
  a[1] = Number(prompt(" b = "));
  b = a.slice();
  let strResult = String(a[0]) + " + " + String(a[1]) + " = ";
  add(a);
  strResult += String(a[0]);
  console.log(strResult);
  strResult = String(b[0]) + " * " + String(b[1]) + " = ";
  multiply(b);
  strResult += String(b[0]);
  console.log(strResult);
  return 0;
}
main();

File: ex06/ex06_10/ex06_10.py
# import copy

def add(arr):
  arr[0] += arr[1] 

def multiply(arr):
  arr[0] *= arr[1]

def main():
  a = []
  b = []
  a.append(float(input(" a = ")))
  a.append(float(input(" b = "))) 
  b = a.copy()
  str_result = str(a[0]) + " + " + str(a[1]) + " = "
  add(a)
  str_result += str(a[0])
  print(str_result)
  str_result = str(b[0]) + " * " + str(b[1]) + " = "
  multiply(b)
  str_result += str(b[0])
  print(str_result)
  return 0

if __name__ == "__main__":
  main()